modules/smmsBidAdapter.js

```diff
@@ -0,0 +1,160 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'smms';
+const ENDPOINT_BANNER = '//bidder.mediams.mb.softbank.jp/api/v1/prebid/banner';
```

Comment:
Banner and native endpoints has to be https to be in 3.0 
It would be great if you can update your endpoints to be  https. We have a PR up to update all adapters endpoint

Reply:
yes, I added https as protocol